internal/cli/universal_login.go

```diff
@@ -266,12 +269,18 @@ func showBrandingTemplateCmd(cli *cli) *cobra.Command {
   auth0 ul templates show`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			var template *management.BrandingUniversalLogin
-
 			if err := ansi.Waiting(func() (err error) {
 				template, err = cli.api.Branding.UniversalLogin()
-				return err
+				if err != nil {
+					if mErr, ok := err.(management.Error); ok && mErr.Status() == http.StatusNotFound {
+						return nil
```

Comment:
Would be nice to have logging here to indicate that the template doesn't exist yet.